sos/plugins/docker.py

```diff
@@ -8,10 +8,10 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin, CosPlugin
 
 
-class Docker(Plugin):
+class Docker(Plugin, CosPlugin):
```

Comment:
This should be in a separate class (as the other examples in this file). The base `Docker` class is really abstract since the package can be deployed in different ways on different distributions (see the differing file collection lists in Red Hat vs. Ubuntu).

This is also where you get to define package triggers for the plugin.

Reply:
I see. Thanks!